test/IISIntegration.FunctionalTests/NtlmAuthentationTest.cs

```diff
@@ -22,29 +21,23 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        public Task NtlmAuthentication(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
```

Comment:
https://github.com/aspnet/ServerTests/issues/82. These tests must be run with the CLR runner, not the CoreCLR runner due to issues with HttpClient.